src/pages/SystemFacilitiesPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from "@apollo/client";
+import { Layout, Table } from "antd";
+import React from "react";
+import Header from "src/components/Header";
+import PageLoader from "src/components/loaders/PageLoader";
+import { GetSystemFacilitiesDocument } from "src/graphql/GetSystemFacilities.generated";
+import { useSelectedFacility } from "src/redux/selectors";
+import { WRAPPER_STYLE } from "src/styles/styles";
+import { reportError } from "src/utils";
+
+const { Content } = Layout;
+
+const SystemFacilitiesPage = () => {
+  const facility = useSelectedFacility();
+
+  const { loading, data } = useQuery(GetSystemFacilitiesDocument, {
+    variables: {
+      facilityId: facility.id,
+    },
+    onError: (err) =>
+      reportError(err, "SystemFacilitiesPage/GetSystemFacilitiesDocument"),
+  });
+
+  if (loading) return <PageLoader />;
+
+  return (
+    <Content>
+      <Header title="System Facilities" subtitle="" extra={undefined} />
+      <div style={WRAPPER_STYLE}>
+        <Table dataSource={data?.facility.system.facilities || []} key="id">
+          <Table.Column title="Name" dataIndex="fullName" />
```

Comment:
should this be `name`?

```suggestion
          <Table.Column title="Name" dataIndex="name" />
```